test/metabase/api/collection_test.clj

```diff
@@ -1807,3 +1808,19 @@
                                        (assoc (graph/graph)
                                               :groups {group-id {default-a :write, currency-a :write}}
                                               :namespace :currency)))))))))
+
+(deftest list-collections-instance-analytics-test
+  (mt/test-drivers #{:postgres :h2 :mysql}
```

Comment:
I don't think this needs to be tested with specific drivers, does it? (This is different from testing with different app DBs, which will happen automatically)